widgetsnbextension/package.json

```diff
@@ -29,7 +29,7 @@
     "@jupyterlab/services": "^6.0.0",
     "@lumino/messaging": "^1.3.3",
     "@lumino/widgets": "^1.11.1",
-    "backbone": "1.2.3"
+    "backbone": "1.4.0"
```

Comment:
Just to double check: this `backbone` update was required to grab the new `preinitialize` method? (and then fix the TS error)

Reply:
Yes looks like it's mentioned in the change log on https://backbonejs.org:

![image](https://user-images.githubusercontent.com/591645/121525709-f71edf00-c9f8-11eb-871b-42307bf72787.png)